build/azure-pipelines/package-product.yml

```diff
@@ -0,0 +1,30 @@
+# sqlcmd package pipeline
+
+pr: none
```

Comment:
can we add a trigger based on the product build pipeline per https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#define-a-pipelines-resource ?